src/locales/en.json

```
@@ -99,8 +99,8 @@
   "error": "Error",
   "errors": {
     "something_went_wrong": "Something went wrong. Check the console for details",
-    "all_nodes_offline": "All {crypto} nodes are offline. Try again later",
-    "all_nodes_disabled": "No active {crypto} nodes. Enable at least one"
+    "all_nodes_offline": "No online ADM nodes. Review the node list",
+    "all_nodes_disabled": "No active ADM nodes. Enable at least one"
```

```suggestion
    "all_nodes_offline": "No online {crypto} nodes. Review the node list",
    "all_nodes_disabled": "No active {crypto} nodes. Enable at least one"
```

We use these translations for other blockchain nodes as well. Pass `crypto` variable inside Vue templates.